imagepipeline-base/src/main/java/com/facebook/cache/disk/DiskStorageCache.java

```diff
@@ -593,8 +601,18 @@ private void updateFileCacheSizeLimit() {
   }
 
   public long getSize() {
+    if (!mCacheStats.isInitialized()){
+            maybeUpdateFileCacheSize();
+        }
     return mCacheStats.getSize();
   }
+  
+  public long getCount() {
+    if (!mCacheStats.isInitialized()){
+            maybeUpdateFileCacheSize();
+        }
+    return mCacheStats.getCount();
+  }
```

Comment:
When testing internally, I've found a couple of issues here.

`maybeUpdateFileCacheSize()` should be guarded by `synchronized(mLock)`.

In any case though, I don't think you should need to call that in either of these methods now as we've recently added an in-memory cache and the stats are now initialized from the constructor.